meteor/client/ui/RundownView/RundownTiming/RundownTimingProvider.tsx

```diff
@@ -121,16 +152,21 @@ export const RundownTimingProvider = withTracker<
 	{
 		static childContextTypes = {
 			durations: PropTypes.object.isRequired,
+			lowResDurations: PropTypes.object.isRequired,
 		}
 
 		durations: RundownTimingContext = {
 			isLowResolution: false,
 		}
+		lowResDurations: RundownTimingContext = {
+			isLowResolution: true,
+		}
 		refreshTimer: number
 		refreshTimerInterval: number
 		refreshDecimator: number
 
 		private timingCalculator: RundownTimingCalculator = new RundownTimingCalculator()
+		private lastSyncedTime: number = 0
```

Comment:
Some information what this variable does would be good, like "Last time the `timeupdateSynced` event was sent."